File: Gestor_canciones/src/gestorAplicacion/ElementosLibreria/Mis_artistas.java
package ElementosLibreria;
import java.util.ArrayList;
import Personas.*;
public class Mis_artistas {
    private ArrayList<Artista> artistas;
    private Usuario usuario;
    public Mis_artistas(Usuario usuario) {
    	this.usuario=usuario;
    }
	public ArrayList<Artista> getArtistas() {
		return artistas;
	}
	public void setArtistas(ArrayList<Artista> artistas) {
		this.artistas = artistas;
	}
	public Usuario getUsuario() {
		return usuario;
	}
	public void setUsuario(Usuario usuario) {
		this.usuario = usuario;
	}
    public void agg_artista(Artista artista) {
    	artistas.add(artista);
    }
    public String elim_artista(Artista artista) {
    	boolean presente=false;
    	for (Artista c: artistas) {
    		if (artista==c){
    			presente=true; 
    		}
    
    	}
    	if (presente==true) {
    		artistas.remove(artista);
    		return"artista eliminado de mis artistas";
    	}
    	else {
    		return"ERROR: el artista no está en mis artistas";
    	}
    }
}

File: Phyton/Gestor_canciones/src/gestorAplicacion/elementosLibreria/Mis_artistas.py
class Mis_artistas():
    mis_artistas=[]
    def __init__(self, usuario):
        self._usuario=usuario
        self._artistas=[]
        Mis_artistas.mis_artistas.append(self)
    def getUsuario(self):
        return self._usuario
    def setUsuario(self, usuario):
        self._usuario=usuario
    def getArtistas(self):
        return self._artistas
    def setArtistas(self, artistas):
        self._artistas=artistas
    def agg_artista(self, artista):
        self._artistas.append(artista)
    def elim_artista(self, artista):
        presente=False
        for i in self._artistas:
            if i==artista:
                presente=True
                break
        if presente==True:
            self._artistas.remove(artista)
            return("artista eliminado de mis artistas")
        else:
            return("ERROR: el artista no esta en mis artistas")